music21/musicxml/m21ToXml.py

```diff
@@ -5557,31 +5557,48 @@ def setBend(mxh: Element, bend: articulations.FretBend) -> None:
 
         >>> MEXClass = musicxml.m21ToXml.MeasureExporter
 
-        >>> a = articulations.FretBend()
+        >>> a = articulations.FretBend(bendAlter=interval.Interval(2))
 
         >>> from xml.etree.ElementTree import Element
         >>> mxh = Element('bend')
 
-        >>> MEXClass.setBend(mxh, a)
-        >>> MEXClass.dump(mxh)
+        >>> MEXclass.setbend(mxh, a)
+        >>> MEXclass.dump(mxh)
+        <bend>
+          <bend-alter>2</bend-alter>
+        </bend>
+        >>> a.pre-bend = True
```

Comment:
this test is not being run since `a.pre-bend = True` is  `a.pre  -  bend = True` which is an incorrect statement in Python.

Reply:
```
In [5]: a.pre-bend = True
  Cell In[5], line 1
    a.pre-bend = True
    ^
SyntaxError: cannot assign to expression here. Maybe you meant '==' instead of '='?

```